profiles/package.mask

```diff
@@ -1,3 +1,7 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-02)
+# Last release in 2018, package does not appear to compile properly.
+dev-haskell/hsqml
```

Comment:
That should come as a separate pull request.

Reply:
I'll submit this separately.